examples/ppm_example.py

```diff
@@ -0,0 +1,43 @@
+from pylab import *
+from pyspecdata import *
+
+# make peaks on a frequency axis
+x = nddata(r_[0:8:49152j], "t2")
+x.set_units("t2", "s")
+
+# generate time-domain signal
+signal = 0 * x  # create an array of zeros that's the right shape
+SFO1 = 400.130438262389
+TMS_shift = 12.5
+for A, nu, R in [
+    (0.1, TMS_shift, 0.04),
+    (1, 25, 0.08),
+    (1, 50, 1.2),
+    (1, 20, 0.45),
+    (1, 750, 1.2),
+    (1, 10, 0.08),
+]:
+    nu = SFO1 + nu # it's really unclear why this is done!
```

Comment:
this doesn't make sense to me

Reply:
if the above represent a table of offsets, why add in SFO1 here?

Reply:
I do not know why I thought I needed this, because it is fine without it. I will remove it.